tests/test_smoke.py

```diff
@@ -58,7 +58,7 @@
 # id.
 test_id = str(uuid.uuid4())[-2:]
 
-LAMBDA_TYPE = '--cloud lambda --gpus A100'
+LAMBDA_TYPE = '--cloud lambda --gpus A10'
```

Comment:
Is this change unintentional?

Reply:
This change was made in master branch. I wrote it while resolving merge conflict. https://github.com/skypilot-org/skypilot/commit/bb6429bab5549dd6b6b63da355173bdfe52d0605